spec/beta/rate_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe EasyPost::Beta::Rate do
+  describe '.list' do
+    it 'retrieve all stateless rates' do
+      stateless_rates = described_class.retrieve_stateless_rate(Fixture.basic_shipment)
+
+      expect(stateless_rates).to all(be_an_instance_of(EasyPost::Rate))
+    end
+
+    describe '.lowest rate' do
```

Comment:
The `describe` block is supposed to define a function or class being tested
```suggestion
    describe '.get_lowest_stateless_rate' do
```